cmd/skaffold/app/cmd/runner.go

```diff
@@ -101,6 +102,14 @@ func runContext(opts config.SkaffoldOptions) (*runcontext.RunContext, []*latest.
 		if err := defaults.Set(config, setDefaultDeployer); err != nil {
 			return nil, nil, fmt.Errorf("setting default values: %w", err)
 		}
+		cwd, err := os.Getwd()
```

Comment:
Could you extract the following code to a utility function and use it here?

https://github.com/GoogleContainerTools/skaffold/blob/35214eb817bff5d541d101d36ad3046be81c183b/pkg/skaffold/trigger/fsnotify/workdir.go#L24-L40